packages/eddsa-poseidon/src/HashFunction.ts

```diff
@@ -0,0 +1,4 @@
+export interface HashFunction {
+    update(data: Buffer): HashFunction
+    digest(): Buffer
```

Comment:
Not sure if this is necessary, but it could be better to use the type provided by the `buffer` package here, instead of the native nodejs Buffer type. You should just add an import.

Reply:
also done